public/app/features/explore/PrometheusListView/RawListItem.tsx

```diff
@@ -115,10 +115,12 @@ const RawListItem = ({ listItemData, listKey, totalNumberOfValues, valueLabels,
   };
 
   // Convert the object back into a string
-  const stringRep = `${__name__}{${attributeValues.map((value) => {
-    // For histograms the string representation currently in this object is not directly queryable in all situations, leading to broken copied queries. Omitting the attribute from the copied result gives a query which returns all le values, which I assume to be a more common use case.
-    return value.key !== 'le' ? `${value.key}="${transformCopyValue(value.value)}"` : '';
-  })}}`;
+  const stringRep = `${__name__}{${attributeValues
+    .filter((value) => value.key !== 'le')
```

Comment:
So I actually fixed another bug here, if you copied a series with an "le" it would leave in the extra comma. But I can't remember why we're excluding the "le", maybe just on the assumption that the user would want the whole histogram? Maybe we just kill this filter now, I bet the problem was introduced by outputting the text representation which wouldn't work in a prom query.

Reply:
I implemented the above change, but you can't see it because github is having issues